rotkehlchen/tests/unit/decoders/test_uniswapv3.py

```diff
@@ -279,3 +280,212 @@ def test_uniswap_v3_swap_received_token2(database, ethereum_manager, eth_transac
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0xdD84Ce1aDcb3A4908Db61A1dFA3353C3974c5a2B']])  # noqa: E501
+def test_uniswap_v3_swap_by_aggregator(database, ethereum_manager, eth_transactions):
```

Comment:
I would also add a test for that heavy swap (and I think there is no need to hard-code all the events for that big swap into the test, we can just query them from chain)

Reply:
The point of the test is to check that decoding is done proper when an AMM is called by a dex aggregator